Terminal.Gui/Views/Scroll/Scroll.cs

```diff
@@ -10,8 +10,11 @@ namespace Terminal.Gui;
 public class Scroll : View
 {
     /// <inheritdoc/>
-    public Scroll ()
+    public Scroll () : this (null) { }
```

Comment:
You can just make that an optional parameter for the same effect.

`public Scroll (Scrollbar? host = null)`

Reply:
I know but for `All View Tester` scenario every `View` sub-class must have a default constructor (parameterless) to being instantiating using reflection. Thanks for remind me.